grid2op/tests/test_score_idf_2023.py

```diff
@@ -71,7 +75,7 @@ def setUp(self) -> None:
             params.LIMIT_INFEASIBLE_CURTAILMENT_STORAGE_ACTION = True
         self.seed = 0
         self.scen_id = 0
-        self.nb_scenario = 2
```

Comment:
and if you keep the "l2rpn_idf_2023" magically you can keep the 2 scenarios and all the assert that were there in the first place.

Reply:
Yes I did this (if I remember well), going back to the env "l2rpn_idf_2023" should fix it + going back to the previous values in the test.

Reply:
To avoid regression (which will happen if we continue to modify the tests like that to "make them pass") I renamed the test files:

- `test_alert_obs_act` (previously `test_alert_feature`)
- `test_AlertReward` (previously `test_alert_score`)
- `test_score_idf_2023_nres` (previously `test_score_idf_2023`)

And the class `AlertReward` should not be modified either.


Neither of these 4 files should be modified in this PR.